src/crypto/verification/request/VerificationRequest.js

```diff
@@ -645,16 +665,7 @@ export class VerificationRequest extends EventEmitter {
 
     _createVerifier(method, startEvent = null, targetDevice = null) {
         if (!targetDevice) {
-            const theirFirstEvent =
-                this._eventsByThem.get(REQUEST_TYPE) ||
-                this._eventsByThem.get(READY_TYPE) ||
-                this._eventsByThem.get(START_TYPE);
-            const theirFirstContent = theirFirstEvent.getContent();
-            const fromDevice = theirFirstContent.from_device;
-            targetDevice = {
-                userId: this.otherUserId,
-                deviceId: fromDevice,
-            };
+            targetDevice = this.estimatedTargetDevice;
```

Comment:
shouldn't this be a function call?

Reply:
It's a getter so it's a secret function call, although tbh given it's already confused one person I'm not sure it's the best place for one. I'd have just gone with `estimateTargetDevice()`, or in fact probably just `targetDevice()` - aren't we always verifying the device that's talking to us, by definition?

Reply:
The code didn't feel very reliable at picking it up, though it can certainly be renamed to `targetDevice`.